CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Wraped actions in tests with `act`, as React requests.
 
 ### Removed
+- [Demo] Removed call MeetingManager.leave() on endMeetingForAll button click. (duplicate)
```

Comment:
Sorry dint understood (duplicate), could we remove it?

Reply:
Sure, updated.